yarn.lock

```diff
@@ -19513,6 +19527,14 @@ micromatch@^4.0.5:
     braces "^3.0.2"
     picomatch "^2.3.1"
 
+micromatch@^4.0.7:
```

Comment:
Shouldn't all of the ^4.0.x versions get merged? I'll see if I can fix this and re-bootstrap.